libraries/CMakeLists.txt

```diff
@@ -52,8 +52,6 @@ set_property(GLOBAL PROPERTY CTEST_CUSTOM_TESTS_IGNORE
   yubico_otp wrap_data wrap info import_rsa import_authkey generate_hmac generate_ec\
   attest pbkdf2 parsing ${_CTEST_CUSTOM_TESTS_IGNORE}")
 
-add_subdirectory( rodeos )
```

Comment:
Can we make it a cmake option to enable rodeos and eosio-tester?

Reply:
Seems kind of heavy weight.  It currently is out of sync and needs to be updated to work correctly, so toggling it with an option is a lie to the end user.

Reply:
If that broken, then shouldn't we remove the directories out of the branch for now?